src/library/ErrorBoundary/index.tsx

```diff
@@ -3,55 +3,71 @@
 
 import { faBug } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useTranslation } from 'react-i18next';
 import { Wrapper } from './Wrapper';
 
+// an example of types of errors, could be unknown
+const error = 403;
```

Comment:
Why is an example error here?

Reply:
can be removed as it's a reminder to me that error can be expanded to different types